### PR TITLE
Add 'get_required_var'

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: doc/backend_vars.asciidoc [hunk 1]
```diff
@@ -19,31 +11,23 @@ IPMI_PASSWORD;;;
...
-|====================
-
 .QEMU backend
 [grid="rows",format="csv"]
 [options="header",cols="^m,^m,^m,v",separator=";"]
 |====================
 Variable;Values allowed;Default value;Explanation
-ADDONS;boolean;0;Enables use of ISO_$i variables
+$k;;;
+$var;;;
```
Comment (aaannz) [inline doc/backend_vars.asciidoc]: Yeah, this is not really useful. I'm taking notes, will update the script later. ( note to #339 )

File: testapi.pm [hunk 2]
```diff
@@ -298,6 +298,19 @@ sub get_var {
...
+
+  get_required_var($variable)
+
+Like get_var but without default. Fails when variable can not be retrieved.
+
+=cut
+
+sub get_required_var {
+    my ($var) = @_;
+    return $bmwqemu::vars{$var} // die "Could not retrieve required var $var";
```
Comment (coolo) [inline testapi.pm]: how about using real words in such statements instead of var?

File: doc/backend_vars.asciidoc (hunk 1 above)
Comment (okurz) [inline doc/backend_vars.asciidoc]: sorry, shouldn't be part of this commit but thanks for noticing :-)

File: testapi.pm (hunk 2 above)
Comment (okurz) [inline testapi.pm]: I just assumed you'd prefer "variable" so I changed it

**Comment by coolo** [comment]
please fix the merge requests

**Comment by okurz** [comment]
@coolo huh?

**Comment by coolo** [comment]
"This branch has conflicts that must be resolved"
